lib/export-process.js

```diff
@@ -0,0 +1,104 @@
+'use strict';
+
+const { Mail } = require('@janiscommerce/mail'); // MailError ??? agregar
+const logger = require('lllog')();
+const InstanceGetter = require('./helpers/instance-getter');
+
+
+class ExportProcess {
+
+	get mustHaveClient() {
+		return true;
+	}
+
+	get mustHavePayload() {
+		return true;
+	}
+
+	get instanceGetter() {
+		if(!this._instanceGetter)
+			this._instanceGetter = this.session.getSessionInstance(InstanceGetter);
+
+		return this._instanceGetter;
+	}
+
+	get exportModel() {
+		if(!this._exportModel)
+			this._exportModel = this.instanceGetter.getModel('export');
+
+		return this._exportModel;
+	}
+
+	get statuses() {
+		return this.exportModel.constructor.statuses;
+	}
+
+	/**
+	 * Do something before Start processing Data
+	 * @async
+	 *
+	 * @param {Object} exportDocument Export Document Before Processing
+	 */
+	async preProcess(exportDocument) {
+		return exportDocument;
+	}
+
+	async process() {
+
+		const exportDocument = this.data;
```

Comment:
Como mencione en el export-create, lo que llegaría como `exportDocument`
```suggestion
		const { exportDocument } = this.data;
```

Reply:
O incluso mejor:

`const { id, ...exportDocument } = this.data.exportDocument;`